eth/tracers/native/call_flat.go

```diff
@@ -224,7 +238,8 @@ func (t *flatCallTracer) GetResult() (json.RawMessage, error) {
 
 // Stop terminates execution of the tracer at the first opportune moment.
 func (t *flatCallTracer) Stop(err error) {
-	t.tracer.Stop(err)
+	t.reason = err
```

Comment:
I'd prefer to keep using `t.tracer.Stop(err)` , remove `t.reason` and use `t.tracer.reason`. Because why have the same field in the outer field, if we have it in the inner?

Reply:
A valid point.
Pushed changes